samples/local_attestation/enc1/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Generate header needed by other enclaves for attestation
+add_custom_command(OUTPUT enc1_pubkey.h
+  DEPENDS public_key_a
+  COMMAND ${CMAKE_SOURCE_DIR}/gen_pubkey_header.sh ${CMAKE_BINARY_DIR}/enc2/enc1_pubkey.h ${CMAKE_BINARY_DIR}/public_a.pem)
+
+add_custom_target(enc1_pubkey_h DEPENDS enc1_pubkey.h)
+add_custom_target(key_info_a DEPENDS public_key_a enc2_pubkey_h)
```

Comment:
Hm... I think I can simplify this but it's too hard to do in a code review. I'll give it a shot and share a commit in a bit.